app/views/course/found.njk

```diff
@@ -1,14 +1,14 @@
 {% extends "_form.njk" %}
 
-{% set title = "Have you found a course to apply to?" %}
+{% set title = "Have you chosen a course to apply to?" %}
```

Comment:
Nice :100:

Reply:
Although, thinking about this some more in context – the purpose of these screens is to choose the provider and course. I wonder if this might be interpreted as "not yet, that's what I'm doing".

The thing to communicate is essentially "do you know what course you're applying to"